R/register-chunk-hook.R

```diff
@@ -0,0 +1,53 @@
+#' Use vcr in vignettes
+#'
+#' @description
+#' `setup_knitr()` registers a `use_cassette` hook so that you map
```

Comment:
I think there was a bit of a mishap when you renamed the script, it's now present twice.